### PR TITLE
1.0: Update usage of new service discovery helper API

### PR DESCRIPTION
**Pull request by kenhys**
Since v1.13.0 new helper API is introduced.
It makes supporting service_discovery easy.

ref. https://github.com/fluent/fluentd/pull/3299

### CODE REVIEW

File: plugin-helper-overview/api-plugin-helper-service_discovery.md [hunk 1]
```diff
@@ -66,5 +74,63 @@ It manages service discovery functionalities such as updating target services an
...
+
+    def send_data(node, chunk)
+      # Send data
+    end
+  end
+end
+```
+
+You can also use helper methods such as `service_discovery_services` or `service_discovery_rebalance` instead of
+`@discovery_manager.services` or `@discovery_manager.services`.
```
Comment (ashie) [inline plugin-helper-overview/api-plugin-helper-service_discovery.md:132]: later one is `@discovery_manager.rebalance`?

**Review by ashie — CHANGES_REQUESTED** [review]

File: plugin-helper-overview/api-plugin-helper-service_discovery.md (hunk 1 above)
Comment (kenhys) [reply plugin-helper-overview/api-plugin-helper-service_discovery.md]: Yes, it should be.

**Comment by kenhys** [comment]
Fixed typo.

**Review by ashie — APPROVED** [review]
LGTM

**Comment by ashie** [comment]
Thanks!

**Pull request merged**